apache-Dockerfile-block-1

```diff
@@ -5,7 +5,7 @@ RUN rm -rf /var/www/html && mkdir -p /var/lock/apache2 /var/run/apache2 /var/log
 # Apache + PHP requires preforking Apache for best results
 RUN a2dismod mpm_event && a2enmod mpm_prefork
 
-RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist
+RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist && rm /etc/apache2/{conf,sites}-enabled/*
```

Comment:
Can we instead move them to `{conf,sites}-enabled.dist` for posterity, like we do with `/etc/apache2/apache2.conf`?